tests/test_2d_operations.cpp

```diff
@@ -16,7 +16,9 @@ using namespace wmtk::tests;
 
 using TM = TriMesh;
 using MapResult = typename Eigen::Matrix<long, Eigen::Dynamic, 1>::MapType;
-using TMOE = decltype(std::declval<DEBUG_TriMesh>().get_tmoe(wmtk::Tuple()));
+using TMOE = decltype(std::declval<DEBUG_TriMesh>().get_tmoe(
```

Comment:
i know i wrote this ugliness, but it's getting even more ugly :sob: . but this is the right thing haha

Reply:
it took me quite some time to figure it out 😅